diff -aurN util-linux-2.24.2.orig/include/pathnames.h util-linux-2.24.2/include/pathnames.h
--- util-linux-2.24.2.orig/include/pathnames.h	2014-05-27 21:02:06.502673192 +0000
+++ util-linux-2.24.2/include/pathnames.h	2014-05-27 21:03:47.676005948 +0000
@@ -26,7 +26,7 @@
 #define	_PATH_DEFPATH_ROOT	"/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin"
 
 #define _PATH_SECURETTY		"/etc/securetty"
-#define _PATH_WTMPLOCK		"/etc/wtmplock"
+#define _PATH_WTMPLOCK		"/run/wtmplock"
 
 #define	_PATH_HUSHLOGIN		".hushlogin"
 #define	_PATH_HUSHLOGINS	"/etc/hushlogins"
@@ -34,7 +34,7 @@
 #define _PATH_NOLOGIN_TXT	"/etc/nologin.txt"
 
 #ifndef _PATH_MAILDIR
-#define	_PATH_MAILDIR		"/var/spool/mail"
+#define	_PATH_MAILDIR		"/var/mail"
 #endif
 #define	_PATH_MOTDFILE		"/etc/motd"
 #define	_PATH_NOLOGIN		"/etc/nologin"
diff -aurN util-linux-2.24.2.orig/login-utils/login.1 util-linux-2.24.2/login-utils/login.1
--- util-linux-2.24.2.orig/login-utils/login.1	2014-05-27 21:02:06.499340042 +0000
+++ util-linux-2.24.2/login-utils/login.1	2014-05-27 21:04:01.752672704 +0000
@@ -275,7 +275,7 @@
 .I /var/run/utmp
 .I /var/log/wtmp
 .I /var/log/lastlog
-.I /var/spool/mail/*
+.I /var/mail/*
 .I /etc/motd
 .I /etc/passwd
 .I /etc/nologin
